lighthouse-core/lib/dependency-graph/base-node.js

```diff
@@ -209,7 +209,7 @@ class BaseNode {
       }
     });
 
-    if (!idToNodeMap.has(this.id)) throw new Error(`Cloned graph missing node ${this.id}`);
+    if (!idToNodeMap.has(this.id)) throw new Error(`Cloned graph missing node`);
```

Comment:
smallest nit in the world of nits....`BACKTICK` -> `COMMA TO THE TOP` since there's no more interpolation

Reply:
Oops. I figured that edit would show up as a suggestion. I just changed it for you :)